dart/dynamics/Joint.h

```diff
@@ -347,6 +354,16 @@ class Joint : public virtual common::Subject
   /// Get upper limit for position
   virtual double getPositionUpperLimit(size_t _index) const = 0;
 
+  /// Gets whether a generalized coordinate is cyclic. Returns true if and only
```

Comment:
We haven't been putting `s` after verb when the comment begin with verb, and I've seen many other projects putting no `s`. Is this matter of personal preference or there is recommended way grammatically?